study-builder/studies/singular/activities/medical-record-file-upload.conf

```diff
@@ -68,6 +68,18 @@
                 },
               ]
             },
+            "tooltipTemplate": {
+               "templateType": "TEXT",
+               "templateText": "$add_participant_incapacitated_tooltip",
+               "variables": [
+                  {
+                     "name": "add_participant_incapacitated_tooltip",
+                     "translations":[
```

Comment:
Shouldn't we use new translations approach? @MocanaAtBroad, what do you think? Or at some point we will have kind of refactoring task for that?